console/console-init/ui/src/modules/iot-device/components/ViewMetaData/MetaDataReviewRow.tsx

```diff
@@ -25,14 +26,12 @@ const styles = StyleSheet.create({
 interface IMetaDataReviewRowProps {
   value: any;
   prevKey?: string;
-  index?: number;
   viewAll?: boolean;
 }
 
 const MetaDataReviewRow: React.FunctionComponent<IMetaDataReviewRowProps> = ({
```

Comment:
Please keep `D` in lower case for folder name , component name.
ViewMetaData==>ViewMetadata
MetaDataReviewRow==>MetadataReviewRow
IMetaDataReviewRowProps==>IMetadataReviewRowProps